pkg/kernel/networkservice/pinggrouprange/common.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023 Cisco and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pinggrouprange
+
+import (
+	"context"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netns"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+	"github.com/networkservicemesh/api/pkg/api/networkservice/mechanisms/kernel"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+
+	"github.com/networkservicemesh/sdk-kernel/pkg/kernel/tools/nshandle"
+)
+
+// See https://github.com/go-ping/ping#linux
+const groupRange = "0 2147483647"
+
+func set(ctx context.Context, conn *networkservice.Connection) error {
+	if mechanism := kernel.ToMechanism(conn.GetMechanism()); mechanism != nil && mechanism.GetVLAN() == 0 {
```

Comment:
Could be used before using applyPingGroupRange(ctx, mech).

This might improve readability because now the function 'set' is doing two things:
1. set ping group range
2. check mech.

So with applying the suggestion the function will do only:
1. set ping group range